src/Italy/Mews.Fiscalizations.Italy/Communication/SoapClient.cs

```diff
@@ -34,7 +35,7 @@ public SoapClient(Uri endpointUri, Func<HttpRequest, Task<HttpResponse>> httpCli
             var response = await HttpClient(httpRequest).ConfigureAwait(continueOnCapturedContext: false);
 
             var soapBody = GetSoapBody(response.Content.Value);
-            return XmlManipulator.Deserialize<TOut>(soapBody);
+            return XmlSerializer.Deserialize<TOut>(soapBody.OuterXml);
```

Comment:
Wasn't this better when it was working with XmlElement? Was it able to automatically handle any encoding before? (I suppose it always needs to be specified now).